docs/our_work/ratings-and-reviews.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "NHS.UK: Automatic Moderation of Ratings & Reviews"
+summary: "Automated moderation of reviews submitted to NHS.UK. "
+origin: ""
+tags: ["NLP", "AI", "ML", "NHS.UK"]
+---
+
+The NHS.UK website receives around a hundred thousand  review every year. These reviews need moderating -- there's [a set of NHS policies which need to be applied to these](https://www.nhs.uk/our-policies/comments-policy/#:~:text=Users%20should%20only%20post%20one,service%20will%20not%20be%20published.) before they can be published.
+
+This project automates much of that work. We use machine learning models - some built by us, some open-source - to make decisions about the different rules which need to be enforced. This reduces the average moderation time from days to seconds, makes for more reliable and consistent moderation, and creates a much more scalable service.
```

Comment:
do we want to point out here that it allows live feedback to be served to the user, so they can make instant changes in response? this is mentioned in the flow but could be worth emphasising it as a further benefit?